keps/NNNN-kep-template/README.md

```diff
@@ -304,7 +304,7 @@ extending the production code to implement this enhancement.
 ##### Integration tests
 
 <!--
-Integration tests are contained in k8s.io/kubernetes/test/integration.
```

Comment:
This wasn't meant to be a link, just a package name :grin:

But using a link of course also works and may be more convenient.